opensrp-anc/build.gradle

```diff
@@ -178,7 +181,7 @@ tasks.withType(Test) {
 
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.3.1'
-    implementation('org.smartregister:opensrp-client-native-form:2.1.19-PREVIEW-SNAPSHOT@aar') {
+    implementation('org.smartregister:opensrp-client-native-form:3.0.2-PREVIEW-SNAPSHOT@aar') {
```

Comment:
Are we using preview because native-forms isn't merged yet? Please link the PR that needs merging for this to change.

Reply:
https://github.com/opensrp/opensrp-client-native-form/pull/649/files

Reply:
yes, https://github.com/opensrp/opensrp-client-native-form/pull/649